src/index.html

```diff
@@ -11,8 +11,61 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The href attribute should link to a CSS file, not a SCSS file. Please change './styles/index.scss' to './styles/index.css'.